docs/tests.rst

```diff
@@ -23,10 +23,95 @@
 INSPIRE Tests
 =======================
 
+How to Write Selenium Tests
+---------------------------
+
+Selenium Test Framework
+~~~~~~~~~~~~~~~~~~~~~~~
+
+INSPIRE's Selenium tests are written using an in-house framework called BAT
+(``inspirehep/bat``). The framework is made of four main components:
```

Comment:
It should, but the documentation build does not know about the code, so it cannot point to it. Best I can do is adding a `:file:` directive, which is a little more semantical but does not point where it should.